.azure/obtemplates/build-winuser.yml

```diff
@@ -13,6 +13,13 @@ jobs:
     ob_sdl_binskim_break: true # https://aka.ms/obpipelines/sdl
     ob_sdl_codeSignValidation_excludes: -|**\*.exe # Disable signing requirements for test executables
   steps:
+  - task: PowerShell@2
+    displayName: Prepare Build Machine
+    target: windows_build_container2
+    inputs:
+      pwsh: true
+      filePath: scripts/prepare-machine.ps1
+      arguments: -InstallXdpSdk -ForContainerBuild
```

Comment:
Why do we need the `-ForContainerBuild` option now, when it wasn't needed before?

Reply:
Because XDP dependencies become must items for any winuser build?